src/hotspot/share/nmt/mallocHeader.cpp

```diff
@@ -24,15 +24,14 @@
  */
 #include "precompiled.hpp"
 
-#include "services/mallocHeader.inline.hpp"
+#include "nmt/mallocHeader.inline.hpp"
 
+#include "nmt/mallocSiteTable.hpp"
```

Comment:
There should be no newline between these two includes. If you make another round of changes I think it would be good to get rid of it.